container_images.mk

```diff
@@ -6,21 +6,21 @@ ETCD_IMAGE = k8s.gcr.io/etcd@sha256:b960569ade5f37205a033dcdc3191fe99dc95b15c679
 ETCD_IMAGE_NAME = etcd
 ETCD_IMAGE_VERSION = 3.2.18
```

Comment:
Shouldn't we want to upgrade the other images to reflect what's defaulted with this version of Kubernetes ? https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md

Reply:
Very good point, that's a blocker, and something that wasn't considered in #824 I believe.

Reply:
etcd get upgraded for kubernetes1.12 so I will do another PR for development/2.1 to upgrade etcd